src/analyze/analyze.c

```diff
@@ -2401,7 +2401,7 @@ static int parse_argv(int argc, char *argv[]) {
                                        "Option --root is only supported for cat-config and verify right now.");
 
         /* Having both an image and a root is not supported by the code */
-        if(arg_root && arg_image)
+        if (arg_root && arg_image)
```

Comment:
This change should be part of the "analyze: support discrete images for verify verb" commit, rather than the follow-up. The easiest way to do it without getting tangled in git, is to undo it as a new commit that gets --fixup to the "analyze: changed manager properties so that full paths are not required for image and root verification" commit, rebase, then apply it again but this time do a fixup to the right one, and rebase again